static/frontEndLayout.js

```diff
@@ -17,6 +15,66 @@ var showFlash = function(message) {
   setTimeout(hideFlash, 1500);
 };
 
+var setElementDisplay = function(name, i, j, display) {
+    var id = name + i + ',' + j;
+    var element = document.getElementById(id);
+    element.style.display = display;
+    return element;
+};
+var hideElement = function(name, i, j) {
+    return setElementDisplay(name, i, j, "none");
+};
+var showElement = function(name, i, j) {
+    return setElementDisplay(name, i, j, "block");
+};
+
+var toolTip = function(msg, id) {
+    var span = document.createElement("SPAN");
+    span.classList.add("tooltiptext");
+    var txt = document.createTextNode(msg);
+    span.appendChild(txt);
+    var container = document.getElementById(id);
+    container.classList.add("tooltip");
+    container.appendChild(span);
+};
+var buyBtn = document.getElementById("buya");
+var sowBtn = document.getElementsByClassName("sowPlantButton");
+var gettingStarted = function() {
+    toolTip("Buy a potato seed", "buya");
```

Comment:
You want to be sure your tutorial makes sense if we change the game a bit.  I think it's a safe bet that your first moves will always be "Buy seed 'a'" and "Plant seed 'a'".  So hard-coding `"buya"` is fine.  But it may not be a potato.  Using `GAME_CONFIG.seeds.a.name` for the text here will fix that.